Makefile

```diff
@@ -3,24 +3,27 @@ ROOT_DIR                        := $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
 
 VENDOR_SRC                      := $(ROOT_DIR)/vendors
 VENDOR_DIR                      := $(ROOT_DIR)/poetry/core/_vendor
+VENDOR_TXT                      := $(VENDOR_DIR)/vendor.txt
 POETRY_BIN                      ?= $(shell which poetry)
 
-
 .PHONY: vendor/lock
-vendor/lock:
-    # regenerate lock file
+vendor/lock: $(VENDOR_LOCK)
+	# regenerate lock file
 	@pushd $(VENDOR_SRC) && $(POETRY_BIN) lock
```

Comment:
Would `$(POETRY_BIN) lock --no-update` make sense here? (if and when it becomes available, of course)
https://github.com/python-poetry/poetry/pull/3034

Reply:
I guess the question is if this is done after a change is made to the pypt or we are simply reloacking I guess.

Reply:
Not sure what I need, what makes sense. I was a bit lost in this corner. Anyway it didn't seem to make my use case easy. My use case is: I want to change only 1 vendored dependency. And the things I had tried would trigger changes for more than 1 vendored dependency. I should give it another go some day so that I can give better comments here.